lib/config/types.ts

```diff
@@ -358,6 +358,21 @@ export type UpdateType =
   | 'bump'
   | 'replacement';
 
+// These are the update types which can have configuration
+export const updateTypesOptions = [
```

Comment:
why not using same case for both? typescript should be smart enough to detect right reference